documentation/modules/about-cold-warm-migration.adoc

```diff
@@ -20,6 +20,15 @@
 
 Cold migration is the default migration type. The source virtual machines are shut down while the data is copied.
 
+[NOTE]
```

Comment:
Trying to simplify the big note into a paragraph, check if it is possible to add it in a similar format-
```suggestion
.VMware migrations
In cold migrations, where a package manager cannot be used during the migration, {project-short} does not install the daemon `qemu-guest-agent` on the migrated VMs. This has some impact on the functionality of the migrated VMs, but overall, they are still expected to function.
To enable {project-short} to automatically install `qemu-guest-agent` on the migrated VMs, ensure that your package manager can install the daemon.
For the fallback option, use your preferred automated or manual procedure to install `qemu-guest-agent` manually.
```